local_planner/src/nodes/planner_functions.cpp

```diff
@@ -39,6 +40,8 @@ void processPointcloud(
               distance < histogram_box.radius_) {
             // subsampling the cloud
             PolarPoint p_pol = cartesianToPolar(toEigen(xyz), position);
+            h_min = std::min(h_min, p_pol.z);
```

Comment:
I like that solution :) maybe we could use a bit more descriptive names such as z_FOV_angle_min or similar?